### PR TITLE
SSE support for IE 8+ and Edge

### PR DESCRIPTION
**Pull request by aounhaider1**
fixes https://github.com/eclipse/smarthome/issues/1657
Signed-off-by: Aoun Bukhari bukhari@itemis.de

### CODE REVIEW

**Comment by kaikreuzer** [comment]
@aounhaider1 You'll have to add the new 3rd party lib to the about.html.

FTR: I have created CQ 12065 for the eventsource.js.

**Comment by kaikreuzer** [comment]
Good news, we have checkin approval!
@aounhaider1 Could you please rebase your branch, we are then good to merge.

**Comment by aounhaider1** [comment]
Rebased.

File: extensions/ui/org.eclipse.smarthome.ui.paper/about.html [hunk 1]
```diff
@@ -93,6 +93,13 @@
 		This bundle includes the UI-Sortable library. The sources can be found at
 		<a href="https://github.com/angular-ui/ui-sortable">github/</a> and are made available under the 
 		<a href="about_files/UISORTABLE_LICENSE">MIT License</a>. 
-		</p>      
+		</p>  
+		    
+        <p>
```
Comment (kaikreuzer) [inline extensions/ui/org.eclipse.smarthome.ui.paper/about.html]: hm, could you resolve tabs vs. spaces in this file as it is currently mixed?

**Pull request merged**